docs/resource-schemas.md

```diff
@@ -321,10 +311,92 @@ have the following values:
 | `cross-platform` | Runs natively on iOS, Android, and/or desktop. |
 | `web`            | Runs on the web.                               |
 
+## Design kit schema
+
+Design kits contain elements, components, guidelines and wireframes for various design tools as the
+design counterpart to coded assets. Design kits are indexed in the
+[carbon-platform](https://github.com/carbon-design-system/carbon-platform) repository
+[`carbon.yml`](https://github.com/carbon-design-system/carbon-platform/blob/main/services/web-app/data/carbon.yml)
+data file.
```

Comment:
Can we add a little more detail here about the usage of design kits in the resource schema versus in the libraries schema? I think the intent is to "define" the design kits in the resource schema top-level key and then to reference zero or more of those in each library schema if a design kit contains the designs for components within that library. But I'm kind of reading between the lines and might not have that correct.

Reply:
@jdharvey-ibm your understanding of the intent is correct:

- a design kit is a top-level resource
- a library is also a top-level resource, and can reference 0 to many design kits for the design kits that contain designs for components in that library

I'll make a pass to call that out more.